### ISSUE TITLE
ClassCastException when deserializing a NULL JSONObject

### ISSUE THREAD

**Issue by Maxr1998** [afollestad/ason#11]
Stacktrace:
```
java.lang.ClassCastException: org.json.JSONObject$1 cannot be cast to com.afollestad.ason.AsonArray
    at com.afollestad.ason.AsonSerializer.deserialize(AsonSerializer.java:182)
    at com.afollestad.ason.Ason.deserialize(Ason.java:374)
```

A fix would be checking if `result == JSONObject.NULL` [here](https://github.com/afollestad/ason/blob/master/src/main/java/com/afollestad/ason/Ason.java#L149) and returning default value.

**Comment by afollestad** [comment]
Now fixed in release 1.4.4.

Thanks for helpful reports!

**Comment by Maxr1998** [comment]
Thanks for the fast updates! 😄
I'm glad I can help.
